dashboard/config/scripts/ocean.script

```diff
@@ -0,0 +1,15 @@
+hidden false
+has_lesson_plan true
+
+stage 'Ocean'
+level 'Ocean 1'
+level 'Ocean 2'
+level 'Ocean 3'
+level 'Ocean 4'
+level 'Ocean 5'
+level 'Ocean 6'
+level 'Ocean 7'
+level 'Ocean 8'
+level 'Ocean 9'
+level 'Ocean 10'
+level 'Ocean 11'
```

Comment:
Actually, I'd really love it if we could name our levels for content rather than number, because I think they will likely change around a bit as we iterate on the flow and ordering.  I think it would make sense to name them for the modes they currently implement.

Reply:
+1

Reply:
Done